client/src/cmdhficlass.c

```diff
@@ -4425,12 +4425,14 @@ static int CmdHFiClassConfigCard(const char *Cmd) {
         print_config_cards();
     }
 
-    if (ccidx > -1 && ccidx < 14) {
+    if (ccidx > -1 && ccidx < 13) {
```

Comment:
better to use ARRAYLEN to make sure index is inside available configs.

Reply:
Will change and submit